area_detector_handlers/tests/test_eiger.py

```diff
@@ -0,0 +1,16 @@
+import h5py
+import numpy as np
+
+from area_detector_handlers.tests.conftest import select_handler
+
+
+@select_handler("AD_EIGER")
```

Comment:
Should we have the same test for `AD_EIGER2`? They may diverge in the future, and we won't catch it.